### PR TITLE
Remove deprecation warning for Rails 5.0

### PR DESCRIPTION
**Pull request by pschambacher**
/cc @etagwerker 

Implemented the fix proposed in #412

### CODE REVIEW

File: lib/database_cleaner/active_record/truncation.rb [hunk 1]
```diff
@@ -27,7 +27,7 @@ def database_cleaner_view_cache
 
       def database_cleaner_table_cache
         # the adapters don't do caching (#130) but we make the assumption that the list stays the same in tests
-        @database_cleaner_tables ||= tables
+        @database_cleaner_tables ||= (::Rails::VERSION::MAJOR >= 5 ? data_sources : tables)
```
Comment (BenMorganIO) [inline lib/database_cleaner/active_record/truncation.rb]: I don't think the parenthesis are needed here... Just style policing :p

**Comment by BenMorganIO** [comment]
:+1: for this!

File: lib/database_cleaner/active_record/truncation.rb (hunk 1 above)
Comment (doits) [inline lib/database_cleaner/active_record/truncation.rb]: Changing this to `::ActiveRecord::VERSION::MAJOR >= 5` should make the tests pass.

**Comment by spickermann** [comment]
+1

**Comment by pschambacher** [comment]
Removed brackets and changed Rails for ActiveRecord. It's now green.

**Comment by etagwerker** [comment]
@pschambacher Thanks! :beers:

**Comment by jasnow** [comment]
Will try it out when the gem is released.

**Comment by BenMorganIO** [comment]
I feel like a tiny version bump would be nice. Would help clean up my tests :)

**Pull request merged**